core/services/pipeline/graph_test.go

```diff
@@ -122,41 +118,38 @@ func TestGraph_TasksInDependencyOrder(t *testing.T) {
 	err := g.UnmarshalText([]byte(DotStr))
 	require.NoError(t, err)
 
-	u, err := url.Parse("https://chain.link/voter_turnout/USA-2020")
-	require.NoError(t, err)
-
 	answer1 := &MedianTask{
 		BaseTask:      NewBaseTask("answer1", nil, 0, 2),
-		AllowedFaults: 1,
+		AllowedFaults: "",
```

Comment:
Is 1 allowed fault the default?

Reply:
The default is `(number of task inputs) - 1`